kart/point_cloud/checkout.py

```diff
@@ -29,6 +47,12 @@ def reset_wc_if_needed(repo):
                 continue
             shutil.copy(lfs_path, wc_tiles_dir / tilename)
 
+            try:
+                args = ["git", "add", dataset.path]
```

Comment:
this is adding the dataset path to the index for each _tile_ right? should it be un-indented by one level? Or possibly run it once with all point-cloud dataset paths as args.

should this have `stdout=subprocess.DEVNULL` and probably stderr too?

Reply:
Unindented by one level, silenced stdout. Left stderr intact.